### PR TITLE
Add syllabus

### PR DESCRIPTION
**Pull request by matthew-brett**

### CODE REVIEW

File: syllabus.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Data
+****
+
+* the structure of raw brain imaging data;
+* loading, manipulating, showing and saving brain images;
+* coordinate systems and transforms for brain images;
+* working with multiple time courses.
+
+********
+Concepts
```
Comment (jbpoline) [inline syllabus.rst]: fMRI analsysis concepts ?

File: syllabus.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  predicted FMRI signal using a hemodynamic response function; modeling the
+  hemodynamic response;
+* estimating multiple regression models; hypothesis testing on multiple
+  regression models; the General Linear Model as generalization of multiple
+  regression;
+* inference on maps of statistics; correction for multiple comparisons;
+  family-wise error; false discovery rate.
+
+****************
+Working practice
```
Comment (jbpoline) [inline syllabus.rst]: Working practice for reproducible analyses ?

Comment (matthew-brett) [reply syllabus.rst]: Um - I have the feeling lately that reproducible gets too much emphasis.  The key thing is quality, where reproducibility follows.   Can you think of a section title that captures that?

**Comment by matthew-brett** [comment]
How about the titles now?

**Comment by jbpoline** [comment]
they look good !

**Pull request merged**